pkgs/development/python-modules/seaborn/default.nix

```diff
@@ -49,8 +49,16 @@ buildPythonPackage rec {
   ] ++ lib.optionals (!stdenv.hostPlatform.isx86) [
     # overly strict float tolerances
     "TestDendrogram"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # Requires internet connection, and, when it's available without darwin
+    # sandbox, throws:
+    #   OSError: [Errno 30] Read-only file system: '/homeless-shelter'
+    "test_load_dataset_string_error"
   ];
 
+  # Avoid using MacOSX backend to prevent crashing in darwin sandbox
+  MPLBACKEND="Agg";
```

Comment:
Please export in preCheck, ideally conditional on darwin

Reply:
Why

Reply:
Agg the correct backend for every platform, should be used for tests, can't hurt the build.

Reply:
Yes, Agg should be used everywhere. The issue is that Linux backends know how to fallback to it gracefully, and macOS one for some reason can not.